.github/workflows/deploy.yml

```diff
@@ -119,28 +119,35 @@ jobs:
           TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
           TWINE_USERNAME: qiskit
         run: twine upload dist/qiskit*
-  gpu-build:
-    name: Build qiskit-aer-gpu wheels
+  gpu-build-cuda11:
+    name: Build qiskit-aer-gpu-cu11 wheels
     runs-on: ubuntu-latest
     steps:
+      - name: Maximize build space
+        uses: easimon/maximize-build-space@master
```

Comment:
Are we running out of space with the newer cuda builds?

Reply:
yes I observed out of space error without this setting. I tested with some parameters here and I set the safety reserved space